packages/docs/docs/sidekicks-quickstart.md

```diff
@@ -27,212 +34,193 @@ At the end of this quickstart you will:
 
-_TODO: Matt: right now the .env file is in the .gitignore which means they would have to create it. thoughts on the best route here? don't want to set ppl up to accidentally commit secrets._
+You can also download the code directly from [the source](https://github.com/fixie-ai/fixie-sidekick-template). While you're there, give us a star! 🦊
 
-### c) Deploy Sidekick to Fixie
+## Step 2: Build the Sidekick code
 
-The moment of truth has arrived! Let's deploy our sidekick to Fixie!
+> TODO(matt): We need to set up a public Corpus ID that the template Sidekick can
+> use out of the box without the user having to create one. I am prioritizing this!
 
-#### Deploying for Development (`fixie serve`)
+The Sidekick is implement in TypeScript with AI.JSX, so you need to
+build it before it can be deployed. To do this, in the `fixie-sidekick-template`
+directory, run:
+  
+```terminal
+$ npm install
+$ npm run build
+```
+The resulting JavaScript code should now be in the `dist/` subdirectory.
 
-When developing or updating a Sidekick, we most likely will be making many changes and testing things as we go. In this case, we want
-to deploy things as quickly as possible.
+## Step 3: Deploy the Sidekick
 
-From the directory where you have your sidekick code, execute the following in your terminal:
+In the `fixie-sidekick-template` directory, simply run:
 
 ```terminal
-npx @fixieai/fixie@latest serve
+$ fixie deploy
 ```
 
-This uses the latest version of the fixie CLI and serves everything from your machine. For example, if our Fixie user is "sarah" and she is deploying a sidekick named "sidekick-acme", then we would expect to see something like this:
+This will deploy the Sidekick to the Fixie cloud service. It takes a couple of
+minutes, but once the process is done, you will see a link to the Sidekick's
+page on the Fixie Console. For example:
 
 ```terminal
-sarah@computer sidekick-acme % npx @fixieai/fixie@latest serve
-🦊 Serving agent sarah/sidekick-acme...
-🌱 Starting local agent process on port 8181...
-🌱 Running: npx --package=@fixieai/sdk fixie-serve-bin --packagePath ./dist/index.js --port 8181
-🌱 Agent stdout: AI.JSX agent listening on http://0.0.0.0:8181.
-
-🦊 Creating new agent sarah/sidekick-acme...
-🚇 Starting tunnel process...
-🥡 Serving agent at https://3fa8367c27e1337.lhr.life
-🥡 Revision a1a29ee1 was deployed to https://console.fixie.ai/agents/sarah/sidekick-acme
+❯ fixie deploy
+🦊 Deploying agent mdw/fixie-sidekick-template...
+🦊 Creating new agent mdw/fixie-sidekick-template...
+⠋  🚀 Deploying... (hang tight, this takes a minute or two!)
+✔ Agent fixie-sidekick-template is running at: https://console.fixie.ai/agents/mdw/fixie-sidekick-template
 ```
 
-As we make changes to our sidekick, we can simply stop serving our sidekick with `Control+C` and then serve up our new changes with the `serve` command as we did above.
+<img src={Step3Profit} alt="Step 3: Profit." width="300"/>
```

Comment:
```suggestion
<img src={Step3Profit} alt="" width="300"/>
```

For a18y, `alt` should be `''` for "purely decorative" content.

Reply:
done